terraform/modules/jenkins/iam.tf

```diff
@@ -50,6 +50,7 @@ data "aws_iam_policy_document" "jenkins_fargate_policy_document" {
       "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/TDRJenkinsNodeLambdaRoleProd",
```

Comment:
Has `terraform fmt` been run on this file

Reply:
No, I keep forgetting :cry: I've run it now.